samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -401,6 +443,16 @@ static void radio_modulated_tx_carrier_duty_cycle(uint8_t mode, uint8_t txpower,
 	nrfx_timer_enable(&timer);
 }
 
+#if CONFIG_NRF21540_FEM
+static void nrf21540_active_delay_calculate(void)
+{
+	uint32_t ramp_up_time = nrf_radio_modecnf0_ru_get(NRF_RADIO) ?
+					FAST_RAMP_UP_TIME : RAMP_UP_TIME;
+
+	nrf21540_active_delay = ramp_up_time - NRF21540_PG_TX_TIME_US;
+}
```

Comment:
This function could be also added to the nrf21540.h API.

Reply:
It can be. Added